tools/packaging/kernel/configs/fragments/whitelist.conf

```diff
@@ -12,3 +12,6 @@ CONFIG_CRYPTO_DEV_SP_PSP
 CONFIG_CRYPTO_DEV_CCP
 CONFIG_HAVE_NET_DSA
 CONFIG_NF_LOG_COMMON
+CONFIG_MANDATORY_FILE_LOCKING
+CONFIG_ARM64_UAO
```

Comment:
Should it belong to somewhere ARM specific?

Reply:
@bergwolf, you're totally right!